Network/Network.cpp

```diff
@@ -600,8 +656,63 @@ namespace WPEFramework
                 strncpy(iarmData.gateway, gateway.c_str(), 16);
                 strncpy(iarmData.primarydns, primarydns.c_str(), 16);
                 strncpy(iarmData.secondarydns, secondarydns.c_str(), 16);
-                iarmData.isSupported = true;
+                iarmData.isSupported = false;
+
+                RFC_ParamData_t param;
+                if (getRFCConfig("Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.Network.ManualIPSettings.Enable", param))
```

Comment:
Change it like,

if (Utils::getRFCConfig("Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.Network.ManualIPSettings.Enable", param))